netlify-tutorial.md

```diff
@@ -96,37 +98,47 @@ Follow the instructions below to configure continuous deployment. This will auto
 
         Copy the Redis URI from the `Service Overview` page in the Aiven console and paste it in as the `REDIS_URI` value.
 
-        Example URI below:
+        The Redis URI will be in this form:
         ```
         rediss://user:password@host:port
         ```
 
 
 1. Deploy the site:
 
-    Go to the **Deploys** section in Netlify and trigger the deployment by clicking the `Trigger deploy` -button.
+    Go to the **Deploys** section in Netlify and trigger the deployment by clicking the `Trigger deploy` button.
 
-    **NOTE:** The database migrations and seed are done during the Netlify deployment process. Make sure you have the PostgreSQL database up and running when deploying to achieve a working environment on the first try.
+    > **Note:** The database migrations and seed are done during the Netlify deployment process. Make sure you have the PostgreSQL database up and running when deploying to achieve a working environment on the first try.
      
 1. Visit the deployed site:
 
     Open the site from the Netlify admin user interface or from the command line with:
 
-    ```
+    ```shell
      netlify open --site
     ```
+    
+    > **Tip** Whrn the site starts up, choose **Recipes** at the top left of the window to go to the main part of the app.
```

Comment:
There is a typo here, it should be `When the site starts up`

Reply:
Thanks for spotting that !!